docs/querying/sql.md

```diff
@@ -337,8 +337,8 @@ String functions accept strings, and return a type appropriate to the function.
 |`UPPER(expr)`|Returns expr in all uppercase.|
 |`REVERSE(expr)`|Reverses expr.|
 |`REPEAT(expr, [N])`|Repeats expr N times|
-|`LPAD(expr, length[, chars])`|Returns a string of "length" from "expr" left-padded with "chars". If "length" is shorter than the length of "expr", the result is "expr" which is truncated to "length". If either "expr" or "chars" are null, the result will be null.|
-|`RPAD(expr, length[, chars])`|Returns a string of "length" from "expr" right-padded with "chars". If "length" is shorter than the length of "expr", the result is "expr" which is truncated to "length". If either "expr" or "chars" are null, the result will be null.|
+|`LPAD(expr, length[, chars])`|Returns a string of `length` from `expr` left-padded with `chars`. If `length` is shorter than the length of `expr`, the result is `expr` which is truncated to `length`. The result will be null if either `expr` is null, or `chars` is either null or empty.|
```

Comment:
Sorry I missed this before, but I tried `lpad` and `rpad` with mysql and postgres since they aren't standard functions. If the padding is an empty string, mysql returns `null` whereas postgres returns the original string. In the past, we've matched the behavior of postgres (e.g., https://github.com/apache/druid/pull/9488#discussion_r391092673), so it'd be good to match the postgres behavior for `lpad` and `rpad` as well.

Reply:
upvote for preferring postgres behavior to mysql

Reply:
Actually, looking closer this will need changed here too https://github.com/apache/druid/blob/master/core/src/main/java/org/apache/druid/math/expr/Function.java#L1917 since it is treating the value as null if the pad is null. If we fix the issue there and check if pad is null or empty we can avoid entering the `StringUtils` function entirely (though we should still fix the `StringUtils` method)